packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,703 @@
+| `authenticatorId`  | String | The Authenticator's unique identifier.                 |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+Returns a [Authenticator](#Authenticator-object).
+
+#### Usage example
+
+This request would set the `status` of the specified Authenticator to `INACTIVE`.
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate"
+```
+
+##### Response
+
+```json
+{
+    "type": "security_key",
+    "id": "aut1nd8PQhGcQtSxB0g4",
+    "key": "webauthn",
+    "status": "INACTIVE",
+    "name": "Security Key or Biometric",
+    "created": "2020-07-26T21:16:37.000Z",
+    "lastUpdated": "2020-07-26T22:01:46.000Z",
+    "_links": {
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "methods": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/methods",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "activate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/activate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Authenticator Admin API object
+
+The Authenticator Admin API only involves one object, which is the Authenticator.
+
+### Authenticator object
+
+#### Authenticator Properties
+
+The Authenticator model defines the following properties:
+
+| Property      | Type                                                            | Description                                                                     |
+| ------------- | --------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| `_links`      | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this object                                                  |
+| `created`     | String (ISO-8601)                                               | Timestamp when the Authenticator was created.                                   |
+| `id`          | String                                                          | A unique identifier for the Authenticator.                                                |
+| `key`         | String                                                          | Unique string that identifies the Authenticator.                                |
```

Comment:
key may not be unique in the future. When we have a catalog in place, ideally an admin should be able to add multiple authenticators with same catalog key to an org and then be able to configure them independently (example - 2 okta verify authenticators where one has FIPS enabled and one doesn't and different users are assigned to use one or the other).